docs/modules/ROOT/pages/explanations/sli.adoc

```diff
@@ -0,0 +1,39 @@
+= Service Level Indicators
+
+This page explains and defines Service Level Indicators (SLI) that are being measured continuously by the platform.
+
+== SLI: Uptime
+
+Uptime conditions::
+- Redis responds to 95% of "ping" requests within a 1-minute interval.
+  The requests are originating from within the same cluster.
+
+Downtime conditions::
+- Redis is not responding to "ping" requests within a 1-minute interval.
+
+[NOTE]
+====
+- Instances in downtime caused by customer error or abuse do not count towards downtime in SLAs.
+- Some health conditions may cause a complete downtime, but may not be treated as "down".
+====
+
+== SLI: Health
+
+Degraded conditions::
+- Redis responds between 1% and 95% of "ping" requests within a 1-minute interval.
+- The configured storage volume is full.
+- Redis continues to serve read operations, but denies write operations.
+- Instance is being automatically updated to a minor or patch version.
+- Instance is being manually upgraded to a major version.
+- Instance is being reconfigured by the customer (for example storage expansion).
+- The platform itself is in maintenance mode and causes maintenance-related downtimes (for example Node drains).
```

Comment:
Not sure if we want to treat maintenance and reconfigurations as "degraded health". Maybe it should be just excluded from uptime?

Reply:
Yes, that should be excluded.